app/src/main/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenEvents.kt

```diff
@@ -27,3 +28,7 @@ data class RegistrationFacilityUserLocationUpdated(val location: LocationUpdate)
 data class RegistrationFacilityLocationPermissionChanged(val result: RuntimePermissionResult) : UiEvent {
   override val analyticsName = "Registration:Facility Selection:Location Permission Changed to $result"
 }
+
+data class RegistrationFacilityConfirmed(val facilityUuid: UUID) : UiEvent {
+  override val analyticsName: String = "Registration:Confirm Facility Selection"
```

Comment:
Can we change the `analyticsName` to be as an action that has happened?

Ex: `Registration:Confirmed Facility Selection`.